app/controllers/consolidations/AssociateDucrSummaryController.scala

```diff
@@ -71,13 +73,16 @@ class AssociateDucrSummaryController @Inject()(
         case ACCEPTED =>
           Redirect(routes.AssociateDucrConfirmationController.displayPage())
             .flashing(FlashKeys.MUCR -> mucrOptions.mucr)
-        case _ => handleError("Unable to submit Association Consolidation request")
+        case _ =>
+          MDC.put("MUCR", mucrOptions.mucr)
```

Comment:
What's this doing?

Reply:
I've seen/used MDC used to add a variable into log messages before.  But isn't this normally done at the beginning of the request handler?  Not just before you log.

Reply:
There is default filter which would add all information from `HeaderCarrier` to `MDC`. But here we would like to have additional information about MUCR to be better searchable in kibana.

Reply:
So what does the output of that warning log line below look like?  You are already adding the MUCR and DUCR to the log message.  I'll have to come and have a chat tomorrow.

Reply:
Yeah, I want to see log too

Reply:
Is it will add just a column on the kibana to have possibility to see/filter/find by MUCR/DUCR?

Reply:
I kibna
* message: Unable to submit Association Consolidation request: MUCR XXXX and DUCR YYY
* ducr: YYY
* mucr: XXXX

Reply:
I think I get it now.   Kibana keys are the result of keys in MDC?   Seems a bit clunky, but if that's what's required.